### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by leo8a**

### CODE REVIEW

**Comment by kpouget** [comment]
thanks for the PR @leo8a, PR looks good to me, 

but I'll let @dagrayvid handle it, I'm not familiar with this repo :)

File: docs/recipes.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ in the BuildConfig template to populate arguments:
 The `driverContainer:` section describes how to build the driver-container an
 extensive list of options is listed here: <https://github.com/openshift/enhancements/pull/357>
 
-An SpecialResource can also have an dependency, a dependency is expressed again
+An SpecialResource can also have a dependency, a dependency is expressed again
```
Comment (dagrayvid) [inline docs/recipes.md]:
```suggestion
A SpecialResource can also have a dependency, a dependency is expressed again
```

**Comment by dagrayvid** [comment]
Thanks @leo8a , the fixes look good to  me, I just commented with one minor typo fix that can be added along with this. Note that we are rewriting the README in #78.

**Pull request merged**